checks/vulnerabilities.go

```diff
@@ -105,7 +105,7 @@ func HasUnfixedVulnerabilities(c *checker.CheckRequest) checker.CheckResult {
 	vulnIDs := osvResp.getVulnerabilities()
 	if len(vulnIDs) > 0 {
 		c.Dlogger.Warn("HEAD is vulnerable to %s", strings.Join(vulnIDs, ", "))
-		return checker.CreateMinScoreResult(CheckVulnerabilities, "unfixed OSV detected")
+		return checker.CreateMinScoreResult(CheckVulnerabilities, "existing vulnerabilities detected")
```

Comment:
maybe add number of vulnerabilities and vuln ids in message, more information will be nice here?

X vulnerabilities detected: VULN_ID_1, VULN_ID_2

Reply:
the details are logged in the line above https://github.com/ossf/scorecard/blob/main/checks/vulnerabilities.go#L107.
The message is just the summary `reason`.

Reply:
the details are logged in the line above https://github.com/ossf/scorecard/blob/main/checks/vulnerabilities.go#L107
The message `existing ...` is just the high-level summary we show in the `reason` column of the results.